caleb-lab/test/test.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
```

Comment:
Again, this test module is riddled with comments that should be left out. That being said, you won't lose points for this lab for it, but we'll get stricter next labs. Also, I see you have lots of questions, so I'll try to answer them.

**TODO FOR LOST POINTS**
Instructions say ``You will need to write tests that guarantee that the files are logged in order.``

Here's a look at Abigail's test module to understand how to test properly that your outputted arrays are the same.